### PR TITLE
[Docs] Sync mmcls mmedit docstring 2.0

### PR DESCRIPTION
**Pull request by hanrui1sensetime**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Update mmcls mmedit docstring for dev-1.x branch.

## Modification

docs and mmdeploy/codebase

## BC-breaking (Optional)

None.

## Use cases (Optional)

None.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by RunningLeon** [comment]
@hanrui1sensetime pls. fix lint

File: mmdeploy/codebase/mmcls/deploy/classification_model.py [hunk 1]
```diff
@@ -28,6 +28,8 @@ class End2EndModel(BaseBackendModel):
         class_names (Sequence[str]): A list of string specifying class names.
```
Comment (RunningLeon) [inline mmdeploy/codebase/mmcls/deploy/classification_model.py:28]: remove this argument

File: mmdeploy/codebase/mmcls/deploy/classification.py [hunk 2]
```diff
@@ -116,6 +118,8 @@ class Classification(BaseTask):
         deploy_cfg (Config): Deployment config file or loaded Config
             object.
         device (str): A string represents device type.
+        experiment_name (str, optional): The experiment name used to create
+            runner. Defaults to 'Classification'.
```
Comment (grimoire) [inline mmdeploy/codebase/mmcls/deploy/classification.py:122]: Add docstring for build_data_preprocessor and get_visualizer

File: mmdeploy/codebase/mmedit/deploy/super_resolution_model.py [hunk 3]
```diff
@@ -129,9 +131,10 @@ def forward(self,
         it is an image.
 
         Args:
-            lq (torch.Tensor): The input low-quality image of the model.
-            test_mode (bool): When test_mode is set, the output is evaluation
-                result. Otherwise it is an image. Default to `False`.
+            inputs (torch.Tensor): The input tensors
```
Comment (grimoire) [inline mmdeploy/codebase/mmedit/deploy/super_resolution_model.py:134]: Add docstring for build_super_resolution_model.

**Review by RunningLeon — APPROVED** [review]
LGTM

**Review by grimoire — APPROVED** [review]

**Pull request merged**